File: java/workspace/day10/src/com/jp/javaexer/Person.java
package com.jp.javaexer;

public class Person {
	public static void main(String[] args) {
		Person p = new Person();
		System.out.println(p.age);
	}
	public int age;
	public String name;
	public Person(){
		age = 18;
	}
}

File: react.js/react_redux/08_src_最终版/redux/reducers/person.js
import {ADD_PERSON} from '../constant';
const initState =[{id:'001',name:'hjp',age:20}]
/* export default personReducer = (preState=initState,action) => {
    const {type,data} = action;
    switch (type) {
        case ADD_PERSON:
            return [data,...preState]
        default:
            return preState
    }
} */
export default function personReducer(preState=initState,action){
    //console.log('@@@')
    const {type,data} = action;
    switch (type) {
        case ADD_PERSON:
            return [data,...preState]
        default:
            return preState
    }
}